### ISSUE TITLE
Is there a way to show system tray as two rows?

### ISSUE THREAD

**Issue by DavidJoyDong** [valinet/ExplorerPatcher#2229]
# Is there a way to show system tray as two rows?

taskbar in one row but system tray shows two rows.

a way in this video https://www.youtube.com/watch?v=QEMDlvuOQmU

put it simply
go regedit `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StuckRects3` and change `00000028_FE` to `02`
but it's not working now.

![image](https://github.com/valinet/ExplorerPatcher/assets/47232992/ae25f24a-9d95-415c-847a-7032b41e7561)

**Comment by Amrsatrio** [comment]
New Win11 taskbar? No. Old Win10 taskbar? Yes, you can do it in the same way as how you do it in previous Windows versions (unlocking the taskbar and resizing).

**Comment by Vipin2003** [comment]
For more simple way, Use Winaero Tweaker (old version) as it provides 3 ways.

**Comment by caccamo88** [comment]
My Win10 taskbar has 2 rows for both. It is one reason because I am using ExplorerPatcher. So switch to Win10 style

**Comment by DinoMC** [comment]
> New Win11 taskbar? No. Old Win10 taskbar? Yes, you can do it in the same way as how you do it in previous Windows versions (unlocking the taskbar and resizing).

In the actual Win10 taskbar (or maybe I'm confusing it with win7 ?), if you dragged up very slightly to make the taskbar higher, it would go into a slightly larger size than default with ONE row of tasks/windows but TWO rows of icons in the notification area.

Just tried with ExplorerPatcher in Win10 mode and if I make it bigger it instantly double in size and has 2 rows of both tasks and icons. Am I doing something wrong or is the thing I'm talking about not implemented ?

**Comment by DavidJoyDong** [comment]
> if you dragged up very slightly to make the taskbar higher, it would go into a slightly larger size than default with ONE row of tasks/windows but TWO rows of icons in the notification area. 

I really love the design you're talking about, but does it actually exist? I've forgotten.

**Comment by pyrates999** [comment]
I just set my entire taskbar to 2 rows instead of 1.  Then the system tray icons are also shown as 2 rows.  I run windows 10 though, so this could be different windows 11.

**Comment by DinoMC** [comment]
> I really love the design you're talking about, but does it actually exist? I've forgotten.

I'm 100% sure it existed (in win7 or 10 or both) because it took me a while to figure out how it worked. 
Sometimes I would resize my taskbar and be back to 1 row of icons, so I'd resize it again and this time it'd have 2 rows. 
Eventually I figured out that I just had to go more slowly cause the "2 row of icons" was in between the size 1 and 2 of the taskbar (basically size 1.5) and it felt great to finally be able to set it up more easily :)

**Comment by dgtwzd1** [comment]
I ran into this same Windows 11 system-tray two-row problem and ended up making a small standalone/source-included utility for it.

Disclosure: I made this:
https://github.com/dgtwzd1/systray-wrap-doubler

It makes Windows 11 notification-area/system-tray icons use a compact two-row layout beside the clock/date area.

It is not an ExplorerPatcher feature request or a registry-only tweak; it is a separate Windows shell customization tool. The repo is MIT licensed and includes source/docs, so if any ExplorerPatcher/Windhawk-style project wants to inspect or adapt the approach, it is there.
